### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by preciz**
Thanks for reviewing :wave: .

### CODE REVIEW

File: lib/stream_data.ex [hunk 1]
```diff
@@ -698,7 +698,7 @@ defmodule StreamData do
 
   ## Examples
 
-  Let's build a generator that returns a binary around 25% of times and a
+  Let's build a generator that returns a binary around 25% of times and an
```
Comment (bryanhuntesl) [inline lib/stream_data.ex]: *of the time and an integer around 75% of the time.*

Comment (bryanhuntesl) [reply lib/stream_data.ex]: everything else is good

Comment (preciz) [reply lib/stream_data.ex]: Thanks, I have updated it. :+1:

**Comment by whatyouhide** [comment]
Thank you @preciz 💟

**Pull request merged**